proxy/contracts/predeployed/LockAndDataForSchainERC20.sol

```diff
@@ -43,19 +54,37 @@ contract LockAndDataForSchainERC20 is PermissionsForSchain {
         // solium-disable-previous-line no-empty-blocks
     }
 
+    /**
+     * @dev Allows ERC20Module to send (mint) ERC20 tokens to LockAndDataForSchainERC20.
```

Comment:
to send mint ERC20 tokens **FROM** LockAndDataForSchainERC20

Reply:
fixed